modules/integration/tests-integration/tests-backend/src/test/resources/org/wso2/identity/integration/test/rest/api/server/application/management/v1/update-claim-configuration.json

```diff
@@ -0,0 +1,13 @@
+{
+  "claimConfiguration": {
+    "subject": {
+      "claim": {
+        "uri": "http://wso2.org/claims/username"
+      },
+      "includeUserDomain": false,
+      "includeTenantDomain": false,
+      "useMappedLocalSubject": true,
+      "mappedLocalSubjectMandatory": true
+    }
+  }
+}
```

Comment:
add new line at the end, do the same for all applicable files

Reply:
addressed with https://github.com/wso2/product-is/pull/17441/commits/4d4a7463fec036d1c6ad94c4014f053bbd61c4e8